packages/medusa-plugin-sanity/src/api/routes/hooks/sanity.js

```diff
@@ -0,0 +1,27 @@
+export default async (req, res) => {
+  try {
+    const sanityService = req.scope.resolve("sanityService")
+
+    //TODO: req sys in Sanity
+    const sanityType = req.body.sys.contentType.sys.id
+    const entryId = req.body.sys.id
```

Comment:
Looks like sanity uses different ids than sys.id; 

Please make sure to update this and make sure that it works :)

Reply:
https://vuuzletvph.com/what-is-the-secret-of-nft-tokens-and-why-they-are-bought-for-millions-of-dollars/

Kindly check this out.Have a great day ahead!

Reply:
https://vuuzletvph.com/heads-up-vuco-bounty-programs-are-coming-your-way/